runtime/tests/org.eclipse.core.tests.harness/META-INF/MANIFEST.MF

```diff
@@ -8,11 +8,18 @@ Export-Package: org.eclipse.core.tests.harness,
  org.eclipse.core.tests.session,
  org.eclipse.core.tests.session.samples
 Require-Bundle: org.junit,
- org.eclipse.test.performance;resolution:=optional,
- org.eclipse.core.runtime
+ org.eclipse.test.performance,
```

Comment:
I'm missing a 3.13 version increase if new API is added here...

Reply:
Done with https://github.com/eclipse-platform/eclipse.platform/pull/57.